combat-trainer.lic

```diff
@@ -494,15 +494,15 @@ class LootProcess
       return
     end
 
-    case bput("stow #{item}", 'You pick up', 'You get', 'You need a free hand', 'There isn\'t any more room', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory', 'The .* is not designed to carry anything', 'rapidly decays away', 'cracks and rots away')
+    case bput("stow #{item}", 'You pick up', 'You put', 'You get', 'You need a free hand', 'needs to be tended to be removed', 'There isn\'t any more room', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory', 'The .* is not designed to carry anything', 'rapidly decays away', 'cracks and rots away')
```

Comment:
Should there be a common "stow matches" somewhere, since we have these two blocks here, and probably more in other scripts?

Reply:
> Should there be a common "stow matches" somewhere, since we have these two blocks here, and probably more in other scripts?

Yes, I've got a second PR that I'm testing now to refactor this stow loot block to use DRCI methods. Great minds think alike =)